### PR TITLE
Initial version of Develop branch

### PR DESCRIPTION
**Pull request by habib535**

### CODE REVIEW

File: src/Search/wwwroot/Search/css/style.css [hunk 1]
```diff
@@ -19,12 +14,91 @@
...
+.search-ranking-guide  {
+    margin-bottom: 5px;
 }
 
-    .search-instant .paper-search-input span {
-        text-align: left;
-        padding-left: 3.5em !important;
-    }
\ No newline at end of file
+.italic-text {
```
Comment (bouguila) [inline src/Search/wwwroot/Search/css/style.css]: repeated css class

File: src/Search/wwwroot/Search/css/style.css [hunk 2]
```diff
@@ -19,12 +14,91 @@
...
+    border-top: 1px solid #8c8b8b;
+    margin-top: 2px;
+    margin-bottom: 10px;
+    opacity: 0.5;
+    -webkit-opacity: 0.5;
+    width: 50%;
+    margin-left: 0px;
+}
+
+.italic-text {
```
Comment (bouguila) [inline src/Search/wwwroot/Search/css/style.css]: css classes should be preceded by app name

File: src/Search/wwwroot/Search/viewmodels/SearchAll.html [hunk 3]
```diff
@@ -1,19 +1,22 @@
 ﻿<link rel="import" href="/sys/vaadin-combo-box/vaadin-combo-box.html">
-<link rel="import" href="../elements/pagination-element/pagination-element.html" />
+<link rel="import" href="/Search/elements/pagination-element.html" />
 
 <template>
     <template is="dom-bind">
         <template is="dom-if" if="{model.SearchTypes.length)}}">
```
Comment (bouguila) [inline src/Search/wwwroot/Search/viewmodels/SearchAll.html]: missing { on if statement

File: src/Search/wwwroot/Search/css/style.css (hunk 2 above)
Comment (warpech) [inline src/Search/wwwroot/Search/css/style.css]: Also, normally you don't want the class names to represent how it looks. You want them to represent what are they used for. In this case, `.search-ranking-step-title`, or so.

File: src/Search/wwwroot/Search/viewmodels/SearchAll.html (hunk 3 above)
Comment (warpech) [inline src/Search/wwwroot/Search/viewmodels/SearchAll.html]: and unneccessary closing `)`

**Comment by habib535** [comment]
Ping @bouguila

**Comment by bouguila** [comment]
looks good now!

**Pull request merged**